tests/ui/sse2.rs

```diff
@@ -20,6 +20,6 @@ fn main() {
                 "SSE2 was not detected as available on an x86 platform");
     }
     // check a negative case too -- allowed on x86, but not enabled by default
-    assert!(cfg!(not(target_feature = "avx2")),
-            "AVX2 shouldn't be detected as available by default on any platform");
+    assert!(cfg!(not(target_feature = "avx512f")),
+            "AVX512 shouldn't be detected as available by default on any platform");
```

Comment:
hmm, if we add a v4 platform this is not the case... why not name a really outlandish target feature, like `"avx512vp2intersect"`? it is now _fully_ deprecated in Intel CPUs, i.e. none of their newer SKUs support them, no matter what segment, so we can expect it to never make it into a platform default. ever.

Reply:
Sure, although I wonder, does this need to be a real feature at all? This test runs on non-x86 targets too, and it seems not to mind if I put something like `"ferris_wheel"` in there either.

Reply:
I put in my silly one, but I can switch to your deprecated feature if preferred for some reason.

Reply:
No that's great. :3